mkdocs.yml

```diff
@@ -90,6 +92,9 @@ markdown_extensions:
   # Plugins: This is a plugin that enables the search engine
   #in the markdown file.
 
+  # this option removes the edit buttton
+edit_uri: ""
```

Comment:
we would rather add this `edit_uri: blob/main/docs/` than eliminate it

Reply:
okay, done this